src/stories/Library/banner/banner.scss

```diff
@@ -0,0 +1,65 @@
+@mixin banner-height {
+  height: 500px;
+  @include media-query__medium {
+    height: 810px;
+  }
+}
+
+@mixin banner-text-margin {
+  margin-bottom: $s-md;
+  @include media-query__medium {
+    margin-bottom: $s-xl;
+  }
+}
```

Comment:
Not sure if a mixin for this is necessary. 

suggestion for naming: banner-text-spacing 

Alternatively just use gap with grid/flex? 

up to you though.

Reply:
I personally liked it, but I'm also open to using the grid if you think it's better.

Reply:
Changed my opinion as it will place the arrow incorrectly if there is no describtion